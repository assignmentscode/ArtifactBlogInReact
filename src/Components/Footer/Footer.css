body, html {
    font-weight: 10;
    margin: 0px;
  }
  .content-width {
    width: 70%;
    margin: auto;
  }
  footer {
    background: #000;
    color: #fff;
    position: relative;
    width:100vw;
    height: 5vw;
    bottom: 0;
  }

  .footer p {
    font-size: 0.8em;  
  }