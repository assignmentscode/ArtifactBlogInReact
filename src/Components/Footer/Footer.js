import React, { Component } from 'react';
import './Footer.css';

class Footer extends Component {
    render(){
        return(
            <footer>
                <article className="content-width">
                    &copy;artifact.com 2019
                </article>
            </footer>
        );
  }
}

export default Footer;