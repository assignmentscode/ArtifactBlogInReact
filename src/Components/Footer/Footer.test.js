import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import Footer from './Footer';

describe('Footer', ()=> {
    it('should render a footer', ()=>{
        const tree = renderer.create(<Footer/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
    const wrapper = shallow(<Footer/>);
    it('should have 1 paragraph', ()=> {
        expect(wrapper.find('p').getElements().length).toEqual(1);
    });
});