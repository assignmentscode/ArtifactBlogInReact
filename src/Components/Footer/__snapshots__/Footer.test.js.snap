// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render a footer 1`] = `
<footer
  className="footer"
>
  <p
    className="content-width"
  >
    Â©artifact.com 2019
  </p>
</footer>
`;
